test/jest-unit.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
please remove this file to keep configuration standardized among outstanding work on tests